lib/money/money.rb

```diff
@@ -1,77 +1,107 @@
-require 'bigdecimal'
-require 'bigdecimal/util'
-
 class Money
   include Comparable
 
-  attr_reader :value, :cents
+  attr_reader :value, :cents, :currency
 
   class << self
-    def new(value = 0, _currency = nil)
+    attr_accessor :parser, :default_currency
+
+    def new(value = 0, currency = nil)
```

Comment:
I'm defaulting to `nil` here because we also want to handle an explicit `nil` value coming from the DB's currency column. Airing on the side of caution. In other words, we want
```
Money.new(1, nil) == Money.new(1)
```